Training1/airbnb/src/components/Footer/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Info from "./info";
+import infos from "./fakeData";
+import "./index.css";
+
+const Footer = () => {
+  console.log(infos);
+  return (
+    <div className="footer">
+      <div className="above">
+        {infos.map(({ name, group }) => (
+          <Info names={name} group={group} />
+        ))}
+      </div>
+      <div style={{ background: "gray", height:'2px',width:'1200px', margin:'50px 50px 10px 70px' }}></div>
```

Comment:
@doankhanh222, Please don't use inline-style, Pls using CSS file for all components/elements